lessons/module-3/rest-architecture-and-urls.md

```diff
@@ -23,7 +23,8 @@ By the end of this lesson, you will:
 <section class="call-to-action">
 ### Warmup
 
-Take some time to explore these [free REST APIs](https://github.com/public-apis/public-apis), and practice making requests from your dev tools console. **NOTE:** The APIs you should be exploring should have a value of **No** for **Auth** as well as **CORS**, otherwise you won't be able to make requests from your dev tools. 
+Take some time to explore these [free REST APIs](https://github.com/public-apis/public-apis), and practice making requests from your dev tools console. 
+**NOTE:** The APIs you should be exploring should have a value of **No** for **Auth** as well as **CORS**, otherwise you won't be able to make requests from your dev tools. 
```

Comment:
It would be helpful to include a reminder to test for CORS before starting to use that API. I often find it confusing because some APIs in this list have CORS set to "no," yet students still encounter CORS issues with them.

Reply:
Hm - it may be better to find a few that we know for sure work and simply list those ones out here.

Reply:
@niksseif Made some updates to use only APIs that definitely work!